### PR TITLE
docs(http): add express.js and fastify guides

### PR DESCRIPTION
**Pull request by jyecusch**

### CODE REVIEW

File: src/pages/guides/getting-started/nodejs/expressjs.mdx [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+nitric up
+SUCCESS  Images Built (51s)
+SUCCESS  Configuration gathered (6s)
+SUCCESS
+         API | Endpoint
+         app | https://xxxxxxxxxx.execute-api.us-east-1.amazonaws.com
+Deployed Stack (1m14s)
+```
+
+Go ahead at test your new Express+Nitric app in the cloud, you can start with the API Gateway URL returned by the `up` command.
```
Comment (raksiv) [inline src/pages/guides/getting-started/nodejs/expressjs.mdx:196]:
```suggestion
Go ahead and test your new Express+Nitric app in the cloud, you can start with the API Gateway URL returned by the `up` command.
```

File: src/pages/guides/getting-started/nodejs/expressjs.mdx [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+
+At this point we can stop the running application and try deploying to a cloud provider.
+
+## Deploy to the cloud
+
+This is where the value of Nitric really shows through. You don't need to perform any manual cloud deployment activities or add solutions like Terraform to get this project into your cloud environment, Nitric takes care of that for you.
+
+To perform the deployment we'll create a `stack`, stacks give Nitric the config needed for a specific cloud instance of this project, such as the provider and region.
+
+The new stack command can help you create the stack by following this prompts, in this case we'll use AWS as an example:
```
Comment (raksiv) [inline src/pages/guides/getting-started/nodejs/expressjs.mdx:167]:
```suggestion
The 'new stack' command can help you create the stack by following these prompts, in this case, we'll use AWS as an example:
```

File: src/pages/guides/getting-started/nodejs/expressjs.mdx [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+<Note>
+  You should notice that while the background worker takes 2 seconds to finish,
+  the HTTP request `GET: /` still returns instantly.
+</Note>
+
+At this point we can stop the running application and try deploying to a cloud provider.
+
+## Deploy to the cloud
+
+This is where the value of Nitric really shows through. You don't need to perform any manual cloud deployment activities or add solutions like Terraform to get this project into your cloud environment, Nitric takes care of that for you.
```
Comment (raksiv) [inline src/pages/guides/getting-started/nodejs/expressjs.mdx:163]:
```suggestion
This is where the true value of Nitric shines. You don't need to perform any manual cloud deployment activities or add solutions like Terraform to get this project into your cloud environment, Nitric takes care of that for you.
```

File: src/pages/guides/getting-started/nodejs/expressjs.mdx [hunk 4]
```diff
@@ -0,0 +1,211 @@
...
+Starting new request
+Request processed
+```
+
+<Note>
+  You should notice that while the background worker takes 2 seconds to finish,
+  the HTTP request `GET: /` still returns instantly.
+</Note>
+
+At this point we can stop the running application and try deploying to a cloud provider.
```
Comment (raksiv) [inline src/pages/guides/getting-started/nodejs/expressjs.mdx:159]:
```suggestion
At this point, we can stop the running application and try to deploy it to a cloud provider.
```

File: src/pages/guides/getting-started/nodejs/expressjs.mdx [hunk 5]
```diff
@@ -0,0 +1,211 @@
...
+http(app)
+```
+
+<Note>
+  If you're familiar with Express.js you'll notice this example doesn't call
+  `app.listen`. The Nitric `http` function takes care of that, as well as
+  binding the application to the correct port in each environment.
+</Note>
+
+At this point we're ready to start testing locally. The project comes with a `dev` script by default, let's run that now:
```
Comment (raksiv) [inline src/pages/guides/getting-started/nodejs/expressjs.mdx:80]:
```suggestion
At this point, we're ready to start testing locally. The project comes with a `dev` script by default, let's run that now:
```

File: src/pages/guides/getting-started/nodejs/expressjs.mdx [hunk 6]
```diff
@@ -0,0 +1,211 @@
...
+├── index.js
+├── nitric.yaml
+├── package.json
+├── README.md
+└── yarn.lock
+```
+
+In this structure you'll notice the `functions` folder. By default, this is where Nitric expects the entrypoint code for your application. However, that's just a convention, we can change that to anything else that suits our needs.
+
+For this example let's replace the functions folder with a single entrypoint file called `app.js`.
```
Comment (raksiv) [inline src/pages/guides/getting-started/nodejs/expressjs.mdx:52]:
```suggestion
For this example, let's replace the 'functions' folder with a single entry-point file named app.js.
```

**Review by raksiv — COMMENTED** [review]
Minor adjustments, otherwise great.

**Review by davemooreuws — DISMISSED** [review]
Love it

**Review by davemooreuws — APPROVED** [review]

**Review by raksiv — APPROVED** [review]

**Pull request merged**